src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        // Create three random balls and print their information
+        for (int i = 0; i < 3; i++) {
+            Ball ball = Lottery.getRandomBall();
```

Comment:
You should create an instance of Lottery and call the method here
```suggestion
            Ball ball = lottery.getRandomBall();
```